src/main/java/mate/academy/mybookshop/dto/UpdateBookRequestDto.java

```diff
@@ -22,4 +23,5 @@ public class UpdateBookRequestDto {
     private BigDecimal price;
     private String description;
     private String coverImage;
+    private Set<Long> categoryIds;
```

Comment:
I think it should also be marked with `@NotEmpty` because looks like right now it can be updated with empty set

Reply:
agree! fixed